core/app/models/workarea/content/block_draft.rb

```diff
@@ -29,7 +32,9 @@ def typecast_data!
       end
 
       def to_block
-        content.blocks.build(attributes.except('content_id'))
+        result = Content::Block.instantiate(as_document.except('content_id'))
```

Comment:
TIL `.instantiate` is a thing. Whats the significance over `.new`?

Reply:
Yeah, this was a TIL I learned for me too. Basically like `new`, but assumes the attributes are raw, as-if from the DB.

Reply:
This is important here because `active_segment_ids` are localized.